recipe/meta.yaml

```diff
@@ -0,0 +1,147 @@
+    - {{ cdt('libxfixes-devel') }}            # [linux]
+    - {{ cdt('libxi-devel') }}                # [linux]
+    - {{ cdt('libxkbfile-devel') }}           # [linux]
+    - {{ cdt('libxrandr-devel') }}            # [linux]
+    - {{ cdt('libxrender-devel') }}           # [linux]
+    - {{ cdt('libxscrnsaver-devel') }}        # [linux]
+    - {{ cdt('libxtst-devel') }}              # [linux]
+    - {{ cdt('libxxf86vm-devel') }}           # [linux]
+    - {{ cdt('mesa-libgbm-devel') }}          # [linux and aarch64]
+    - {{ cdt('mesa-libgl-devel') }}           # [linux]
+    - {{ cdt('mesa-libegl-devel') }}          # [linux]
+    - {{ cdt('mesa-dri-drivers') }}           # [linux]
+    - {{ cdt('pciutils-devel') }}             # [linux]
+    - {{ cdt('xcb-util-devel') }}             # [linux]
+    - {{ cdt('xcb-util-image-devel') }}       # [linux]
+    - {{ cdt('xcb-util-keysyms-devel') }}     # [linux]
+    - {{ cdt('xcb-util-renderutil-devel') }}  # [linux]
+    - {{ cdt('xcb-util-wm-devel') }}          # [linux]
+    - {{ cdt('xorg-x11-proto-devel') }}       # [linux]
+    - bison       # [linux]
+    - flex        # [linux]
+    - gperf       # [linux]
+    - jom         # [win]
+    - m2-bison    # [win]
+    - m2-flex     # [win]
+    - m2-gperf    # [win]
+    - m2-patch    # [win]
+    - patch       # [unix]
+    - pkg-config  # [unix]
+    - libtool     # [unix]
+    - cmake
+    - html5lib
+    - ninja
+    - nodejs
+    - perl
+
+  host:
+    - qtbase {{ version }}
+    - qtdeclarative {{ version }}
+    - qtwebchannel {{ version }}
+    # Qt tools Designer is required to compile designer plugin.
+    - qttools {{ version }}
+    - expat {{ expat }}            # [linux]
+    # Qt6 requires at least 2.10.92 for access to FCConfigGetSysRoot()
+    # https://www.freedesktop.org/software/fontconfig/fontconfig-devel/fcconfiggetsysroot.html
+    - fontconfig {{ fontconfig }}  # [linux]
+    - freetype 2                   # [linux]
+    - gtk2 2                       # [linux]
+    - icu {{ icu }}                # [linux]
+    - lcms2 2                      # [linux]
+    # Must match version of protobuf Chromium is building with or else generated messages error out on include.
+    - libprotobuf 3.20.3           # [linux]
+    - libtiff {{ libtiff }}        # [linux]
+    - libxcb 1                     # [linux]
+    - libxkbcommon 1               # [linux]
+    - nspr 4                       # [linux]
+    - nss 3                        # [linux]
+    - openjpeg 2                   # [linux]
+    - dbus {{ dbus }}              # [linux]
+    - minizip 4                    # [linux]
+    - pcre 8                       # [linux and aarch64]
+    # Needs to be in hosts to allow QtPdf to build but is really a build dependency so no need to add to runtime deps.
+    - html5lib 1
+    - poppler 24
+
+  run_constrained:
+    - qt-main >={{ version }},<7
+    - qt >={{ version }},<7
+
+test:
+  commands:
```

Comment:
binary existence checks I find a little too minimal. Any chance of trying to run some [smoke tests](https://github.com/conda-forge/qt-webengine-feedstock/blob/6f0bb73ab82bc812b7477bba6deeab25c2a732fd/recipe/meta.yaml#L150)?